release-notes.md

```diff
@@ -24,6 +33,84 @@
     + [Unit testing](#unit-testing)
     + [Integration tests parallelization](#integration-tests-parallelization)
 
+# v0.4.0
+
+This release has been tracked through the [0.4.0 project](https://github.com/cloud-hypervisor/cloud-hypervisor/projects/4).
+
+Highlights for `cloud-hypervisor` version 0.4.0 include:
+
+### Dynamic virtual CPUs addition
+
+As a way to vertically scale Cloud-Hypervisor guests, we now support dynamically
+adding virtual CPUs to the guests, a mechanism also known as CPU hot plug.
+Through hardware-reduced ACPI notifications, Cloud Hypervisor can now add CPUs
+to an already running guest and the high level operations for that process are
+documented [here](https://github.com/firecracker-microvm/firecracker/blob/master/docs/hotplug.md)
```

Comment:
You meant `https://github.com/cloud-hypervisor/cloud-hypervisor/blob/master/docs/hotplug.md`, right?

Reply:
Oops...Yes. Fixed now.